en/config_rover/differential.md

```diff
@@ -0,0 +1,305 @@
+# Configuration/Tuning (Differential Rover)
+
+This topic provides a step-by-step guide for setting up your [Differential rover](../frames_rover/differential.md).
```

Comment:
FYI, if you don't have the sidebar by default (i.e. in mobile view) it can be useful to have first link back to parent for context. Breadcrumb by default would be better, but this does no harm.